src/ui/public/courier/saved_object/ui/saved_object_save_as_checkbox.html

```diff
@@ -1,7 +1,11 @@
 <div ng-hide="!savedObject.id || savedObject.isSaving">
-  <div ng-hide="!savedObject.isTitleChanged() || savedObject.copyOnSave" class="kuiLocalDropdownWarning kuiVerticalRhythmSmall">
-    In previous versions of Kibana, changing the name of a {{savedObject.getDisplayName()}} would make a copy with the new name. Use the 'Save as a new {{savedObject.getDisplayName()}}' checkbox to do this now.
-  </div>
+  <div
+    ng-hide="!savedObject.isTitleChanged() || savedObject.copyOnSave"
+    class="kuiLocalDropdownWarning kuiVerticalRhythmSmall"
+    i18n-id="common.ui.courier.savedObject.howToSaveAsNewDescription"
+    i18n-default-message="In previous versions of Kibana, changing the name of a {savedObjectName} would make a copy with the new name. Use the 'Save as a new {savedObjectName}' checkbox to do this now."
+    i18n-values="{ savedObjectName: savedObject.getDisplayName() }"
```

Comment:
Could we make 'Save as a new {savedObjectName}' in this string actually a parameter, that's just reading in `common.ui.courier.savedObject.saveAsNewLabel`, since that's the string we are referring to. If that won't work we should at least add a description, like: `'Save as a new {savedObjectName}' refers to common.ui.courier.savedObject.saveAsNewLabel and should be the same text.`